DOCS-7870 made edits (#73)
mulesoft/docs-service-mesh: modules/ROOT/pages/download-and-install-service-mesh.adoc

`curl -Ls http://anypoint.mulesoft.com/servicemesh/xapi/v1/install > asmctl && chmod +x asmctl`

> == Configure Your Environment Variable on Linux Systems

Reviewer: I'd suggest this slight change, although the current version is OK. You might also check with the dev to see if this should be "Linux or macOS Systems".
```suggestion
== Configure Your PATH Variable on Linux Systems
```

Author: Will do!

Reviewer 2: **Unix-like**, or ***nix** IMO: https://en.wikipedia.org/wiki/Unix-like

Author: Tomas said Linux. Will check with him later

Reviewer: Yeah, "Unix-like" or "*nix" would include Linux, macOS, and more rare systems such as BSD or the various old commercial mainframe flavors of Unix. "Unix-like" or "*nix" is the most technically correct option. On the other hand, I'm afraid some macOS and even Linux users might not understand that it applies to them. I'm pretty sure most people using a flavor of Unix more rare than Linux will know that "Linux or macOS" means them too. So, of the options, I like "Linux or macOS" the best, but I leave it up to Priya (and Tomas if she consults with him).

Reviewer: Oh-- I'm foolish. I just realized that Service Mesh probably only runs on Linux, in which case it should just say Linux.

Author: Done! Thanks!

Before you begin installing Anypoint Service Mesh, configure your PATH environment variable to store system-wide values, such as the directories to search for the executable programs.

To configure your PATH variable on Linux:

. Open the .bashrc file in your home directory.
+

> For example, type `cd /home/_your-user-name_/.bashrc)` in a text editor.

Reviewer: If you like it, you can use this slightly simpler version:
```suggestion
For example, type `cd ~/.bashrc)` in a text editor.
```

> For example, type `cd /home/_your-user-name_/.bashrc)` in a text editor.

Reviewer: I think this is incorrect and should be changed as follows. If in doubt, check with the dev.
```suggestion
For example, type `nano ~/.bashrc` to open the file with the nano text editor.
```

Reviewer 2: 👍 I made the same correction in another PR but it was merged while I was commenting 😢

Author: Ok, will do!

Author: Done! Thanks!
